src/main/k/working/tests/eqtest/Makefile

```
@@ -0,0 +1,34 @@
+TOP != git rev-parse --show-toplevel
+include $(TOP)/include.mk
+
+KOMPILED := eqtest-kompiled
+DEFINITION := $(KOMPILED)/definition.kore
+
+$(DEFINITION): eqtest.k $(K)
+	$(KOMPILE) $(KOMPILE_OPTS) $< --syntax-module eqtest
+
+%.eqtest.kore: %.eqtest $(DEFINITION) $(KORE_EXEC)
+	$(KRUN) $(KRUN_OPTS) $< --dry-run
+
+%.output: %.eqtest $(DEFINITION) $(KRUN_TARGETS)

```

`$(KRUN_TARGETS)` needs to be replaced with `$(K) $(KORE_EXEC)`.
```suggestion
%.output: %.eqtest $(DEFINITION) $(K) $(KORE_EXEC)
```